plugins/aea-ledger-ethereum-hwi/aea_ledger_ethereum_hwi/account.py

```diff
@@ -24,8 +24,8 @@
 from typing import Any, Generator, List, NamedTuple, Optional
 
 import rlp
+from aea_ledger_ethereum_hwi.bip32 import h, m
 from aea_ledger_ethereum_hwi.exceptions import HWIError
-from apduboy.lib.bip32 import h, m
```

Comment:
We were using a small chunk of code from `apduboy` package so I removed it completely and ported the code over